README.md

```diff
@@ -66,6 +66,27 @@ The rest of the required binaries are downloaded by the playbook to a temporary
 If we, for some reason, want to skip the download (for example if we already have the required binaries at that location from a previous playbook run), 
 we can set the `run_prereqs` parameter in this way: ``-e run_prereqs=false``
 
+#### 7. The parameters required to support image pull secrets (for ISV operators only)
```

Comment:
why is this only for ` ISV operators only`?
If you using the playbooks in standalone mode with a community operator...would image pull secrets not work?

Reply:
In theory, the playbook role should work for community-operators if they use the same encryption method though I haven't tested it.
Do you think I shall mention both ISV & community-operators here and expand the example of community-operators?

Reply:
I would remove `(for ISV operators only)`